src/Microsoft.DotNet.Wpf/src/PresentationCore/MS/internal/Ink/InkSerializedFormat/InkSerializer.cs

```diff
@@ -281,14 +281,13 @@ private void ExamineStreamHeader(Stream inkdata, out bool fBase64, out bool fGif
             return;
         }
 
-        private static readonly byte[] Base64HeaderBytes
-                                            = new byte[]{(byte)'b',
-                                                        (byte)'a',
-                                                        (byte)'s',
-                                                        (byte)'e',
-                                                        (byte)'6',
-                                                        (byte)'4',
-                                                        (byte)':'};
+        private static ReadOnlySpan<byte> Base64HeaderBytes => [(byte)'b',
+                                                                (byte)'a',
+                                                                (byte)'s',
+                                                                (byte)'e',
+                                                                (byte)'6',
+                                                                (byte)'4',
+                                                                (byte)':'];
```

Comment:
UTF-8 string literals can be used to make this (IMO) easier to read:

```csharp
private static ReadOnlySpan<byte> Base64HeaderBytes => "base64"u8;
```

Reply:
@bgrainger This doesn't make use of [RuntimeHelpers.CreateSpan&lt;T&gt;](https://learn.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.runtimehelpers.createspan?view=net-8.0) yet, maybe once this is improved in the future.

Reply:
I believe `RuntimeHelpers.CreateSpan` is only used when endianness matters (When sizeof(T) > sizeof(byte)). In this case, RuntimeHelpers.CreateSpan is useless so Roslyn does not use it.

It does basically the same thing:
https://sharplab.io/#v2:CYLg1APgAgTAjAWAFBQMwAJboMLoN7LLrHoAOATgJYBuAhgC4CmmcAbOgEqO3ADyAdgBsAngGVStfgB4ARsKYA+dACFaAZ0asALAAluwRuWXzGauOgC8SgNoAKOUwCUAchnOANERLefvv/4DAoPsTF1oPLyCo6JiokKdnNQikWNS01PjGF0Zk9Lz8v0yXVlyCsvyi5y1S8trYypBnAF0Abkj0dooaBmYoNk59ARFxSVkTJVUNbT0eQ2MmNRhLJQAiGXVNLRAVgFcADjaUkjRMLXQAWVtHfHaAX2RboA=

Reply:
Ah, yeah, I've just checked the generated assembly, it is still an RVA in the module. And the endianess thing is correct, forgot about that.

Another time then; more readable code.